boards/feather_m0_express/board_config.h

```diff
@@ -0,0 +1,19 @@
+#ifndef BOARD_CONFIG_H
+#define BOARD_CONFIG_H
+
+#define VENDOR_NAME "Adafruit Industries"
+#define PRODUCT_NAME "Feather M0"
```

Comment:
call this "Feather M0 Express" to distinguish it from plain "Feather M0" ?

Reply:
ok